compiler/one-cmds/one-quantize

```diff
@@ -125,6 +125,12 @@ def _get_parser():
         help=
         "Force MaxPool Op to have the same input/output quantparams. NOTE: This option can degrade accuracy of some models.)"
     )
+    quantization_group.add_argument(
+        '--quant_config',
+        type=str,
+        help=
+        "Path to the quantization configuration file.)"
```

Comment:
Q) end of help has `)`... others too. is there any particular reason?

Reply:
I think they are typos. Thanks for finding.

Reply:
I think fix in anther PR will do.